nixos/modules/services/networking/pdns-recursor.nix

```diff
@@ -21,7 +18,7 @@ let
     else if builtins.isList val then (concatMapStringsSep "," serialize val)
     else "";
 
-  configFile = pkgs.writeText "recursor.conf"
+  configDir = pkgs.writeTextDir "recursor.conf"
```

Comment:
Ah! Clever, I didn't know of this builder.

Reply:
I only recently learned of it from the `powerdns` module :smile: